### ISSUE TITLE
🐛 biome check --log-level=error return ERROR

### ISSUE THREAD

**Issue by akkadaya** [biomejs/biome#7788]
### Environment information

```block
CLI:
  Version:                      2.2.6
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           v22.19.0
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.jsonc
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  true

Workspace:
  Open Documents:               0
```

### What happened?

1. run
```sh
biome check --log-level=error
```
2. returns
```
2025-10-18T11:08:34.050448Z ERROR  Biome couldn't open the file 
"/Users/xxx/Projects/project/biome.json", reason:
Os { code: 2, kind: NotFound, message: "No such file or directory" }
    at crates/biome_fs/src/fs.rs:209 on main
```

### Expected result

It should not throw an issue

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
That's how we decided to print that particular part of the logging, but there's no error in how Biome terminates the program.

Feel free to send a PR to make it a warning or an info log. Or did you have anything else in mind @akkadaya ?

**Comment by akkadaya** [comment]
My apologies, I created the issue quickly without investigating it a bit.

I can see that the issue is with Biome config file name `biome.jsonc`.

When running this command, it doesn't use `biome.jsonc` (with c)
```sh
biome check --log-level=error
```
It says 
> 2025-10-19T08:58:21.321784Z ERROR  Biome couldn't open the file "/Users/xxx/Projects/biome-playground/biome.json", reason:
Os { code: 2, kind: NotFound, message: "No such file or directory" }
    at crates/biome_fs/src/fs.rs:209 on main

But it works fine with any other Biome command I tried.

**Comment by akkadaya** [comment]
Here is a repo to reproduce

https://github.com/akkadaya/biome_playground

**Comment by ematipico** [comment]
Sorry @akkadaya but that's **not a minimal reproduction**.

**Comment by Netail** [comment]
@akkadaya `log-level` includes the logging of internal biome logs. Do you mean to use; `--diagnostic-level=error` instead perhaps, to only show linting **errors**? Or do you mean biome should only throw an error log if no config (json nor jsonc) is found? The log of no json biome config being found, is meant as an indication that; a json config was not found, let's try other file extensions. (Could be helpful for debugging purposes)

**Comment by akkadaya** [comment]
Thanks @Netail I didn't know it was the intended behaviour. In that case it's not a bug but my mistake.
